bundles/core/src/test/resources/schema/adaptive-form-properties.schema.json

```diff
@@ -0,0 +1,322 @@
+{
+  "$id": "classpath:/schema/adaptive-form-properties.schema.json",
```

Comment:
this approach is not scalable. I'm assuming that we will fetch the publicly published schema in future.

Reply:
Yes, this is true for even the VS code plugin. Till the schema is public, we maintain a copy.